sample/build.gradle

```diff
@@ -24,8 +24,7 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:23.0.0'
-
-    compile 'au.com.jtribe.shelly:shelly:0.1.6@aar'
-//    compile project(':shelly')
+    compile 'com.android.support:appcompat-v7:24.2.0'
+    //compile 'au.com.jtribe.shelly:shelly:0.1.6@aar'   //use shelly from jcenter
```

Comment:
can probably just remove this. You won't want to ever pull down the jcenter version for the sample app.